src/batch-client/index.ts

```diff
@@ -566,6 +568,52 @@ export class ZimbraBatchClient {
 			return normalized;
 		});
 
+	public searchConversation = ({
+		conversationId,
+		query,
+		header,
+		timezone,
+		locale,
+		cursor,
+		...options
+	}: SearchConversationOptions) => {
+		let body: any;
+		body = {
+			cid: conversationId,
+			_content: {
+				...(query && { query }),
+				...(header && { header }),
+				...(timezone && { tz: timezone }),
+				...(locale && { locale }),
+				...(cursor && { cursor })
+			}
+		};
+
+		// conditionally include the rest of the parameters
+		forEach(options, (value: any, key: string) => {
+			if (value) {
+				if (key === 'allowableTaskStatus' || key === 'types') {
+					body[key] = value.join(',');
+				} else {
```

Comment:
else on next line

Reply:
The linter requires same line:

`prettier: Replace ⏎↹↹↹↹ with ·`

Reply:
of course it does. Because why be consistent with our zm-x-web repo :P